packages/ra-core/src/controller/create/useCreateController.spec.tsx

```diff
@@ -1,60 +1,49 @@
 import React from 'react';
 import expect from 'expect';
 import { act } from '@testing-library/react';
+import { Location } from 'history';
 
-import { getRecord } from './useCreateController';
+import { getRecordFromLocation } from './useCreateController';
 import { CreateController } from './CreateController';
 import { renderWithRedux } from 'ra-test';
 import { DataProviderContext } from '../../dataProvider';
 import { DataProvider } from '../../types';
 
 describe('useCreateController', () => {
-    describe('getRecord', () => {
-        const location = {
+    describe('getRecordFromLocation', () => {
+        const location: Location = {
+            key: 'a_key',
             pathname: '/foo',
             search: undefined,
             state: undefined,
             hash: undefined,
         };
 
-        it('should return an empty record by default', () => {
```

Comment:
why did you remove this one ?

Reply:
Because that's not the responsability of this function anymore